spec/features/api_access_spec.rb

```diff
@@ -7,12 +7,28 @@ def response_json
   end
 
   describe "api json export" do
-    it 'routes to the all action on default' do
+    it 'exports all organizations as json by default' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.